clients/fides-js/docs/README.md

```diff
@@ -26,9 +26,7 @@ Google Tag Manager, IAB TCF, etc.
 See [Fides](interfaces/Fides.md) for how to use the `window.Fides` JavaScript API, [FidesOptions](interfaces/FidesOptions.md) for supported options to customize it's behavior, and [FidesEvent](interfaces/FidesEvent.md) for how to use `window.addEventListener(...)` to subscribe to
 custom Fides events for real-time updates.
 
-## Table of contents
```

Comment:
Table of contents are removed: https://typedoc-plugin-markdown.org/docs/migration-guide#-breaking-changes

I'm OK with that.